Documentation/unix-instructions.md

```diff
@@ -1,5 +1,5 @@
-Building CoreFX on Linux
```

Comment:
This file is linked in a few places, e.g. build.sh and maybe in coreclr. Did you check if those links are all updated to the new name?